docs/source/dataset_summary.rst

```diff
@@ -82,8 +82,7 @@ SSVEP
    :class:`Nakanishi2015`,9,8,12,15,4.15s,256Hz,1
    :class:`Wang2016`,34,62,40,6,5s,250Hz,1
 
-
-c-VEP
+c-VEP and Burst-VEP
```

Comment:
```suggestion
c-VEP
```

I think burst-VEP is a subtype of c-VEP so we can keep `c-VEP` as section name for simplicity. 

@thijor @sebVelut what do you think?

Reply:
I think it was is to compared to the "Gold code"  in the summary of the dataset Thielen2021

Reply:
Are you talking about the `Codes` column of the tables? I'm talking about the name of the section in the doc

Reply:
The code column should be the code family name indeed, like m-sequence, Gold code, random code, or burst code. The section name I would indeed stick to c-VEP. Many other categories exist indeed, like burst-VEP and r-VEP, though I consider these all to be c-VEP (or broad-band VEP as opposed to steady-state VEP).

Reply:
Ho sorry I thought it was the Codes columns of the table, if it's the section's name, I agree it shoulld be just C-VEP